src/modules/transactions/hooks/list/useGetPendingTransactionsList.ts

```diff
@@ -0,0 +1,32 @@
+import { ITransactionsGroupedByMonth } from '../../services';
+import { IPendingTransactionsRecord } from './useTransactionList';
+
+const usePendingTransactionsList = (
+  homeTransactions: ITransactionsGroupedByMonth[],
+  transactionsPageList: ITransactionsGroupedByMonth[],
+): IPendingTransactionsRecord => {
+  const mergedTransactionsList =
```

Comment:
use:
const mergedTransactionsList = `[... homeTransactions, ... transactionsPageList]` and don`t need a validation